zipkin-server/src/main/java/zipkin/server/ZipkinHttpCollector.java

```diff
@@ -63,6 +80,47 @@
     return validateAndStoreSpans(encoding, Codec.JSON, body);
   }
 
+  @Singleton
+  @Named
+  public static class RiposteSpanIngestEndpoint extends StandardEndpoint<Void, String> {
```

Comment:
This is the Riposte version of the `@RequestMapping(value = "/api/v1/spans", method = POST)` Spring Boot endpoint. I went for the lowest amount of code changes I could reasonably get away with in this PoC, so we just inject `ZipkinHttpCollector` and reuse the available methods, converting between Spring `ListenableFuture` -> java 8 `CompletableFuture`, and Spring `ResponseEntity` -> Riposte `ResponseInfo`.